concepts/solvers.mdx

```diff
@@ -4,22 +4,27 @@ sidebarTitle: "Solvers"
+
+<Note>We assume there is a single solver (called the "leader") performing operations on behalf of the users. The leader can be selected in various ways, such as through an auction or a lottery.</Note>
 
-## Issuing and Sourcing Preconfs
+The reason for considering a single solver is that if a user requests the same intent from multiple solvers, it requires coordination between them to fulfill the intent profitably. For example, if two solvers are trying to fulfil an intent that involves transactions on two separate chains, one solver might manage to source preconfirmations from both chains while the other solver only sources one. In this case, both solvers will have to pay for the preconfirmations to the providers, but only one solver will actually fulfil the user's intent and get paid by the user. Coordination between solvers is out of scope of the mev-commit protocol, as it depends exclusively on third-parties.
 
-Preconfirmations serve various purposes. They can benefit a user seeking to profit from an arbitrage opportunity or simply wanting early confirmation of a payment. Requesting a preconfirmation can be difficult without the sophistication to interact with mev-commit providers and estimate the correct price for a preconfirmation bid. 
+## Sourcing and Handling Preconfirmations
 
-1. The solver accumulates the preconfirmations it receives for each bid and it only sends a preconfirmation to the user if/when it manages to source preconfirmations from all the providers for a bid. In this case, the preconfirmation is 100% guaranteed as it was issued by all the providers.
-2. The solver relays back to the user all the preconfirmations that it receives from the providers. In this case the preconfirmations the user gets are “weak” in the sense that they are tied to specific providers, and not from all the providers as is the case above.
-3.  The solver can immediately send a preconfirmation to the user (if it chooses to do so) even before getting preconfirmations from the providers. In this case, the solver is the one issuing the preconfirmation to the user, and the solver is responsible for sourcing a credible preconfirmation from the providers. In case the solver cannot source the preconfirmation from the providers then it may get penalized by slashing of its collateral.
+The solver leader interacts with its user and the mev-commit protocol by receiving user intents and transforming them into mev-commit bids. The solver then tries to source preconfirmations for the bid on the mev-commit protocol. Once preconfirmations are successfully acquired, there are three main ways the solver can handle them. We describe each of them next.
```

Comment:
we generally haven't used intent language, I'm not against but I would use "signed message". also commitments over preconfs, it doesn't make as much sense to get 5 preconfs